src/ReactLeafletMap/layers/VectorLayer/Feature/index.js

```diff
@@ -213,23 +227,25 @@ class Feature extends React.PureComponent {
     renderShape(style) {
         if (!this.icon) {
             this.icon = new MarkerIcon(this.iconId, style, {
-                iconAnchor: [style.radius, style.radius],
+                iconAnchor: style.radius ? [style.radius, style.radius] : null,
                 onMouseMove: this.onMouseMove,
                 onMouseOut: this.onMouseOut,
                 onMouseOver: this.onMouseMove,
                 onClick: this.onClick
             });
+
+            this.icon.setStyle(style);
         }
 
-        // TODO style memoization
-        if (style) {
+        if (!_.isEqual(this.style, style)) {
```

Comment:
Is _.isEqual necessary?

Reply:
Discussed in https://github.com/gisat-panther/ptr-maps/pull/28